### ISSUE TITLE
getallheaders undefined

### ISSUE THREAD

**Issue by Spreeuw** [postcode-nl/PostcodeNl_Api_WooCommerce#3]
getting this error on PHP 7.3.31
```
 PHP Fatal error: Uncaught Error: Call to undefined function PostcodeNl\AddressAutocomplete\getallheaders() in /PostcodeNl_Api_WooCommerce/src/PostcodeNl/AddressAutocomplete/Proxy.php:31
```
I'm reading this should be available on PHP7.3 even for FPM, but it looks like it doesn't. Perhaps simply reading `$_SERVER` instead suffices too, or using a [polyfill](https://github.com/ralouphie/getallheaders)?

**Comment by BastiaanGrutters** [comment]
Thanks for your bug report. 

This will be addressed in the next release.

**Comment by BastiaanGrutters** [comment]
The fix is included in [1.0.6](https://github.com/postcode-nl/PostcodeNl_Api_WooCommerce/releases)

[Related commit](https://github.com/postcode-nl/PostcodeNl_Api_WooCommerce/commit/6756d84aa55115290ebf5c57539fdd7c0f941211)
